doc/policy/mempool-replacements.md

```diff
@@ -65,7 +65,7 @@ This set of rules is similar but distinct from BIP125.
 
 ## History
 
-* Opt-in full replace-by-fee (without inherited signaling) honoured in mempool and mining as of
```

Comment:
I'd revert this line, the docs and code sometimes [use the British spelling](https://www.grammarly.com/blog/honor-honour), e.g. some funny ones like [QColor warning_colour](https://github.com/bitcoin/bitcoin/blob/master/src/qt/sendcoinsdialog.cpp#L876)